gridstatus/caiso.py

```diff
@@ -198,38 +804,51 @@ def get_load_forecast(self, date, end=None, sleep=4, verbose=False):
             raw_data=False,
             verbose=verbose,
             sleep=sleep,
+            params={"market_run_id": forecast_type},
         )
 
         df = df.rename(
-            columns={"MW": "Load Forecast"},
+            columns={"MW": "Load Forecast", "TAC_AREA_NAME": "TAC Area Name"},
         )
 
-        # returns many areas, we only want one overall iso
-        df = df[df["TAC_AREA_NAME"] == "CA ISO-TAC"]
-
         df = df.sort_values("Time")
 
         df = self._add_forecast_publish_time(
             df,
             current_time,
+            forecast_vintage=forecast_vintage,
+            # TODO(kladar): verify exactly  when other forecast types are published
```

Comment:
Here is the publication schedule for demand load forecasts. The daily forecasts are supposed to be published at 9:10 am. 

<img width="1299" alt="Screenshot 2025-01-19 at 20 29 38" src="https://github.com/user-attachments/assets/32c9b1bd-59fd-4d29-94cd-db9d966e0a06" />